Cargo.lock

```diff
@@ -2351,6 +2526,24 @@ dependencies = [
  "tokio",
 ]
 
+[[package]]
+name = "rad-gpg"
```

Comment:
I think something's weird with this lock file? Can you try resetting the changes and then building as normal?